go.sum

```diff
@@ -6,8 +6,8 @@ github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/gofrs/uuid v3.2.0+incompatible/go.mod h1:b2aQJv3Z4Fp6yNu3cdSllBxTCLRxnplIgP/c0N/04lM=
 github.com/intel-go/cpuid v0.0.0-20181003105527-1a4a6f06a1c6/go.mod h1:RmeVYf9XrPRbRc3XIx0gLYA8qOFvNoPOfaEZduRlEp4=
 github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0/go.mod h1:1NbS8ALrpOvjt0rHPNLyCIeMtbizbir8U//inJ+zuB8=
-github.com/libvirt/libvirt-go v5.2.0+incompatible h1:rWfBxTWWbDxkxfeXV8G3qr3pS1YRKkUo7C0EHhXzfEw=
-github.com/libvirt/libvirt-go v5.2.0+incompatible/go.mod h1:34zsnB4iGeOv7Byj6qotuW8Ya4v4Tr43ttjz/F0wjLE=
+github.com/libvirt/libvirt-go v3.4.0+incompatible h1:Cpyalgj1x8JIeTlL6SDYZBo7j8nY3+5XHqmi8DaunCk=
```

Comment:
Interesting

Reply:
@gbraad yes, interesting. Took me 4-5 hours to figure that out :(

Reply:
Does this also mean we could run this against the older libvirt versions itself?

Reply:
so this will work >= 3.4.0 afaik don't have a machine to test out older version of libvirt for this.

Reply:
I would look at `objdump -T  /usr/local/bin/crc-driver-libvirt | grep LIBVIRT` and see if it has anything newer than 3.4 listed.

Reply:
I think the risk is low to merge and see if this puppy runs. It will run for the current expected functionality, so the only question here is if we can allow binding wit older versions. But, I do not think this hold back a merge. WDYT?

Reply:
Yep, definitely no need to carefully check this before merging this PR, can be checked later.